File: 1.10.c
#include <stdio.h>

/*
 *	cat, showing escape codes
 */

int main()
{
	int c;

	for(c = getchar(); c != EOF; c = getchar())
		if(c == '\t')
			printf("\\t");
		else if(c == '\b')
			printf("\\b");
		else if(c == '\\')
			printf("\\\\");
		else
			putchar(c);
}

File: 1.10.py
#!/usr/bin/env python3

# cat, replacing some characters with escape codes

from fileinput import input
from sys import stdout

white = 0

for line in input():
    for c in line:
        if c == '\t':
            stdout.write('\\t')
        elif c == '\b':
            stdout.write('\\b')
        elif c == '\\':
            stdout.write('\\\\')
        else:
            stdout.write(c)
